data/human/free worlds 2 middle.txt

```diff
@@ -117,7 +117,7 @@ mission "FW Diplomacy 1C"
 		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
 	
 	on offer
-		log "Stopped on Earth with Alondo to talk to Parliament and try to head off another bloody battle. But, they were not willing to negotiate."
+		log "Stopped on Earth with Alondo to talk to Parliament and try to head off another bloody battle. Unfortunately, they were not willing to negotiate."
```

Comment:
Is there any reason for this change?

Reply:
There are two meanings of `but` that could be here.
One is "except for the fact".  "Except for the fact, they were not willing to negotiate" doesn't seem like good grammar.
The other is "on the contrary", which also doesn't work.
However, "Unfortunately" conveys sadness, which is seen later on, and also works alright with the comma.

Reply:
I think Unfortunately sounds better than But. "However" could also be a good fit if one wanted something a bit more neutral.